src/components/Swap/Settings/MaxSlippageSelect.tsx

```diff
@@ -27,6 +27,10 @@ const Custom = styled(BaseButton)<{ selected: boolean }>`
   ${inputCss}
 `
 
+const ExpandoContent = styled(Row)`
```

Comment:
Is this generic enough that it should be another export of Expando.tsx?
I'm not sure if it would be good to share, or make derivative components more confusing. Up to you.

Reply:
no, i think the issue here is more that we can't easily add a bit of styling without creating a new named Styled Component. This use case isn't specific to Expando or unique enough to need to be centralized in the Expando file imo. If anything, i think it shows that we should allow adding margin to`Row`.